httpgrpc/server/server_test.go

```diff
@@ -95,9 +95,10 @@ func TestParseURL(t *testing.T) {
 		err      error
 	}{
 		{"direct://foo", "foo", nil},
-		{"kubernetes://foo:123", "kubernetes://foo:123", nil},
-		{"querier.cortex:995", "kubernetes://querier.cortex:995", nil},
-		{"foo.bar.svc.local:995", "kubernetes://foo.bar.svc.local:995", nil},
+		{"kubernetes://foo:123", "kubernetes:///foo:123", nil},
+		{"querier.cortex:995", "kubernetes:///querier.cortex:995", nil},
+		{"foo.bar.svc.local:995", "kubernetes:///foo.bar.svc.local:995", nil},
+		{"kubernetes:///foo:123", "kubernetes:///foo:123", nil},
```

Comment:
Can we also add `dns:///foo:123` to the list so that we are not k8s specific, but also have another resolver in the list.

Reply:
Done (I copied the line from your PR).